runtime/vm/jvminit.c

```diff
@@ -1868,7 +1868,35 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 				}
 			}
 #endif /* !defined(WIN32) && !defined(J9ZTPF) */
+			/* Parse options related to Large Pages */
+			{
+				IDATA argIndexUseLargePages = FIND_AND_CONSUME_ARG(EXACT_MATCH, MAPOPT_XXUSELARGEPAGES, NULL);
+				IDATA argIndexLargePageSizeInBytes = FIND_AND_CONSUME_ARG(STARTSWITH_MATCH, MAPOPT_XXLARGEPAGESIZEINBYTES_EQUALS, NULL);
+				
+				if (argIndexUseLargePages > argIndexLargePageSizeInBytes) {
```

Comment:
Does this match the RI's behaviour?  Do they treat `-XX:LargePageSizeInBytes=1m -XX:+UseLargePages` differently than `-XX:+UseLargePages -XX:LargePageSizeInBytes=1m`?  Is there verbose output of some sort that shows that?

Reply:
> Is there verbose output of some sort that shows that?
Unfortunately there isn't. We only know that both these options affect both the codecache and objectheap. The design was a piggy-back on how `-Xlp` determines which option to use; the rightmost always wins.